docs/resources/task.md

```diff
@@ -3,25 +3,34 @@
 This resource will:
 
 1. Create cloud resources (machines and storage) for the task.
```

Comment:
💅🏼  The title of this page is **Task Resource** but the nav entry is `iterative_task`. Seems inconsistent

Reply:
Yes, it should probably be `Resource: iterative_task` like in other providers.